Auto update failing for yazi

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [yazi](https://github.com/termux/termux-packages/blob/90331a43bc44dc31dfa9cd96bc4ffffaf46ea2ad/packages/yazi) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating yazi [Current version: 0.2.5]
INFO: package being updated to 0.3.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  749k    0  749k    0     0   985k      0 --:--:-- --:--:-- --:--:-- 1602k

You are about to commit these changes:

--------------------
diff --git a/packages/yazi/build.sh b/packages/yazi/build.sh
index 8cb3d0a63..792b57608 100644
--- a/packages/yazi/build.sh
+++ b/packages/yazi/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://yazi-rs.github.io/
 TERMUX_PKG_DESCRIPTION="Blazing fast terminal file manager written in Rust, based on async I/O"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.2.5"
+TERMUX_PKG_VERSION="0.3.0"
 TERMUX_PKG_SRCURL=https://github.com/sxyazi/yazi/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=aea4a6ebd7f56c5d5fe6afbea143fc98c336d4ccd7852f2e8983c23e205988c4
+TERMUX_PKG_SHA256=0a0c1583accca16759392f258367156a2c36fb0b1d37152b07e1aa5239c531ff
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
--------------------

bump(main/yazi): 0.3.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package yazi exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building yazi for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0   188k      0 --:--:-- --:--:-- --:--:--  191k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1498      0 --:--:-- --:--:-- --:--:--  1498
gpg: Signature made Thu 01 Aug 2024 07:47:57 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24218      0 --:--:-- --:--:-- --:--:-- 24218
100 13320  100 13320    0     0  24214      0 --:--:-- --:--:-- --:--:-- 24174
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1491      0 --:--:-- --:--:-- --:--:--  1492
gpg: Signature made Mon 29 Jul 2024 07:12:42 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  24284      0 --:--:-- --:--:-- --:--:-- 24251
100 13120  100 13120    0     0  24279      0 --:--:-- --:--:-- --:--:-- 24251
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1562      0 --:--:-- --:--:-- --:--:--  1565
gpg: Signature made Wed 31 Jul 2024 06:34:01 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://github.com/sxyazi/yazi/archive/refs/tags/v0.3.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  145k    0  145k    0     0   235k      0 --:--:-- --:--:-- --:--:--  235k
100  749k    0  749k    0     0   977k      0 --:--:-- --:--:-- --:--:-- 4030k
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: syncing channel updates for '1.80.0-x86_64-unknown-linux-gnu'
info: latest update on 2024-07-25, rust version 1.80.0 (051478957 2024-07-21)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'
info: default toolchain set to '1.80.0-x86_64-unknown-linux-gnu'

  1.80.0-x86_64-unknown-linux-gnu installed - rustc 1.80.0 (051478957 2024-07-21)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, you need to source
the corresponding env file under $HOME/.cargo.

This is usually done by running one of the following (note the leading DOT):
. "$HOME/.cargo/env"            # For sh/bash/zsh/ash/dash/pdksh
source "$HOME/.cargo/env.fish"  # For fish
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
    Updating crates.io index
    Updating git repository `https://github.com/notify-rs/notify.git`
 Downloading crates ...
  Downloaded tokio-macros v2.4.0
  Downloaded proc-macro2 v1.0.86
  Downloaded num-conv v0.1.0
  Downloaded time-macros v0.2.18
  Downloaded derive_builder_macro v0.20.0
  Downloaded derive_builder_core v0.20.0
  Downloaded time-core v0.1.2
  Downloaded option-ext v0.2.0
  Downloaded stability v0.2.1
  Downloaded is_terminal_polyfill v1.70.0
  Downloaded quick-error v2.0.1
  Downloaded loop9 v0.1.5
  Downloaded built v0.7.4
  Downloaded bit_field v0.10.2
  Downloaded spin v0.9.8
  Downloaded rustix v0.38.34
  Downloaded jpeg-decoder v0.3.1
  Downloaded onig_sys v69.8.1
  Downloaded lua-src v546.0.2
  Downloaded tiff v0.9.1
  Downloaded luajit-src v210.5.8+5790d25
  Downloaded rav1e v0.7.1
  Downloaded tikv-jemalloc-sys v0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7
  Downloaded yazi-prebuild v0.1.2
  Downloaded linux-raw-sys v0.4.14
  Downloaded syntect v5.2.0
  Downloaded bstr v1.9.1
  Downloaded gimli v0.29.0
  Downloaded object v0.36.1
  Downloaded better-panic v0.3.0
  Downloaded libc v0.2.155
  Downloaded tokio v1.39.2
  Downloaded half v2.4.1
  Downloaded zune-inflate v0.2.54
  Downloaded exr v1.72.0
  Downloaded color_quant v1.1.0
  Downloaded byteorder-lite v0.1.0
  Downloaded bitflags v1.3.2
  Downloaded png v0.17.13
  Downloaded avif-serialize v0.8.1
  Downloaded nom v7.1.3
  Downloaded num-integer v0.1.46
  Downloaded bitstream-io v2.5.0
  Downloaded new_debug_unreachable v1.0.6
  Downloaded num-derive v0.4.2
  Downloaded profiling v1.0.15
  Downloaded aligned-vec v0.5.0
  Downloaded bumpalo v3.16.0
  Downloaded flate2 v1.0.30
  Downloaded tracing-core v0.1.32
  Downloaded tracing v0.1.40
  Downloaded cc v1.1.6
  Downloaded itertools v0.12.1
  Downloaded mlua v0.9.9
  Downloaded indexmap v2.2.6
  Downloaded backtrace v0.3.73
  Downloaded crossbeam-channel v0.5.13
  Downloaded sharded-slab v0.1.7
  Downloaded tracing-subscriber v0.3.18
  Downloaded base64 v0.21.7
  Downloaded quick-xml v0.31.0
  Downloaded tokio-util v0.7.11
  Downloaded clap_builder v4.5.11
  Downloaded flume v0.11.0
  Downloaded lebe v0.5.2
  Downloaded gif v0.13.1
  Downloaded image-webp v0.1.3
  Downloaded fdeflate v0.3.4
  Downloaded qoi v0.4.1
  Downloaded imgref v1.10.1
  Downloaded num-bigint v0.4.6
  Downloaded num-rational v0.4.2
  Downloaded noop_proc_macro v0.3.0
  Downloaded profiling-procmacros v1.0.15
  Downloaded simd_helpers v0.1.0
  Downloaded v_frame v0.3.8
  Downloaded kamadak-exif v0.5.5
  Downloaded base64 v0.22.1
  Downloaded av1-grain v0.2.3
  Downloaded wasm-bindgen-macro-support v0.2.92
  Downloaded crossbeam-utils v0.8.20
  Downloaded rgb v0.8.45
  Downloaded erased-serde v0.4.5
  Downloaded jobserver v0.1.32
  Downloaded home v0.5.9
  Downloaded which v6.0.1
  Downloaded pkg-config v0.3.30
  Downloaded mlua-sys v0.6.1
  Downloaded rustc-hash v2.0.0
  Downloaded num-traits v0.2.19
  Downloaded ordered-float v2.10.1
  Downloaded arc-swap v1.7.1
  Downloaded addr2line v0.22.0
  Downloaded tikv-jemallocator v0.6.0
  Downloaded tracing-appender v0.2.3
  Downloaded overload v0.1.1
  Downloaded nu-ansi-term v0.46.0
  Downloaded filetime v0.2.23
  Downloaded inotify v0.10.2
  Downloaded concurrent-queue v2.5.0
  Downloaded event-listener v4.0.3
  Downloaded block-buffer v0.10.4
  Downloaded typenum v1.17.0
  Downloaded digest v0.10.7
  Downloaded bincode v1.3.3
  Downloaded onig v6.4.0
  Downloaded line-wrap v0.2.0
  Downloaded plist v1.6.1
  Downloaded tokio-stream v0.1.15
  Downloaded clap v4.5.11
  Downloaded clap_complete v4.5.11
  Downloaded time v0.3.36
  Downloaded ratatui v0.27.0
  Downloaded regex-automata v0.4.7
  Downloaded regex-syntax v0.8.4
  Downloaded maybe-rayon v0.1.1
  Downloaded wasm-bindgen-backend v0.2.92
  Downloaded ravif v0.11.9
  Downloaded crossbeam-epoch v0.9.18
  Downloaded crossbeam-deque v0.8.5
  Downloaded rayon-core v1.12.1
  Downloaded rayon v1.10.0
  Downloaded bytemuck v1.16.1
  Downloaded crc32fast v1.4.2
  Downloaded adler v1.0.2
  Downloaded simd-adler32 v0.3.7
  Downloaded miniz_oxide v0.7.4
  Downloaded weezl v0.1.8
  Downloaded zune-core v0.4.12
  Downloaded image v0.25.2
  Downloaded imagesize v0.13.0
  Downloaded mutate_once v0.1.1
  Downloaded derive_builder v0.20.0
  Downloaded getset v0.1.2
  Downloaded vergen-lib v0.1.3
  Downloaded vergen-gitcl v1.0.0
  Downloaded ansi-to-tui v3.1.0
  Downloaded errno v0.3.9
  Downloaded serde-value v0.7.0
  Downloaded globset v0.4.14
  Downloaded equivalent v1.0.1
  Downloaded rustc-demangle v0.1.24
  Downloaded console v0.15.8
  Downloaded signal-hook-tokio v0.3.1
  Downloaded lazy_static v1.5.0
  Downloaded thread_local v1.1.8
  Downloaded tracing-log v0.2.0
  Downloaded inotify-sys v0.1.5
  Downloaded parking v2.2.0
  Downloaded generic-array v0.14.7
  Downloaded md-5 v0.10.6
  Downloaded walkdir v2.5.0
  Downloaded uzers v0.12.0
  Downloaded colorchoice v1.0.1
  Downloaded utf8parse v0.2.2
  Downloaded anstream v0.6.14
  Downloaded clap_derive v4.5.11
  Downloaded clap_complete_nushell v4.5.3
  Downloaded vergen v9.0.0
  Downloaded unicode-width v0.1.13
  Downloaded wasm-bindgen-shared v0.2.92
  Downloaded wasm-bindgen v0.2.92
  Downloaded typeid v1.0.0
  Downloaded fdlimit v0.3.0
  Downloaded async-priority-channel v0.2.0
  Downloaded crypto-common v0.1.6
  Downloaded same-file v1.0.6
  Downloaded clap_complete_fig v4.5.2
  Downloaded anstyle-parse v0.2.4
  Downloaded anstyle-query v1.1.0
  Downloaded anstyle v1.0.7
  Downloaded clap_lex v0.7.1
  Downloaded deranged v0.3.11
  Downloaded num_threads v0.1.7
  Downloaded powerfmt v0.2.0
  Downloaded winnow v0.6.16
  Downloaded toml_edit v0.22.18
  Downloaded toml v0.8.17
  Downloaded serde_json v1.0.121
  Downloaded idna v0.5.0
  Downloaded url v2.5.2
  Downloaded darling_core v0.20.9
  Downloaded syn v1.0.109
  Downloaded proc-macro-error v1.0.4
  Downloaded anyhow v1.0.86
  Downloaded futures-channel v0.3.30
  Downloaded futures-util v0.3.30
  Downloaded signal-hook v0.3.17
  Downloaded crossterm v0.27.0
  Downloaded once_cell v1.19.0
  Downloaded zerocopy v0.7.35
  Downloaded ahash v0.8.11
  Downloaded allocator-api2 v0.2.18
  Downloaded hashbrown v0.14.5
  Downloaded strum_macros v0.26.4
  Downloaded itertools v0.13.0
  Downloaded aho-corasick v1.1.3
  Downloaded memchr v2.7.4
  Downloaded regex v1.10.5
  Downloaded serde_derive v1.0.204
  Downloaded serde v1.0.204
  Downloaded bytes v1.6.1
  Downloaded mio v1.0.1
  Downloaded parking_lot v0.12.3
  Downloaded socket2 v0.5.7
  Downloaded syn v2.0.71
  Downloaded arg_enum_proc_macro v0.3.4
  Downloaded arrayvec v0.7.4
  Downloaded minimal-lexical v0.2.1
  Downloaded wasm-bindgen-macro v0.2.92
  Downloaded zune-jpeg v0.4.13
  Downloaded tracing-attributes v0.1.27
  Downloaded mlua_derive v0.9.3
  Downloaded instant v0.1.13
  Downloaded unicode-bidi v0.3.15
  Downloaded tinyvec v1.6.0
  Downloaded unicode-normalization v0.1.23
  Downloaded darling v0.20.9
  Downloaded validator v0.18.1
  Downloaded futures-task v0.3.30
  Downloaded futures v0.3.30
  Downloaded cassowary v0.3.0
  Downloaded castaway v0.2.3
  Downloaded ryu v1.0.18
  Downloaded compact_str v0.7.1
  Downloaded bitflags v2.6.0
  Downloaded thiserror-impl v1.0.62
  Downloaded thiserror v1.0.62
  Downloaded futures-core v0.3.30
  Downloaded log v0.4.22
  Downloaded mio v0.8.11
  Downloaded unicode-segmentation v1.11.0
  Downloaded pin-project-lite v0.2.14
  Downloaded toml_datetime v0.6.8
  Downloaded form_urlencoded v1.2.1
  Downloaded fnv v1.0.7
  Downloaded ident_case v1.0.1
  Downloaded strsim v0.11.1
  Downloaded darling_macro v0.20.9
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded validator_derive v0.18.1
  Downloaded dirs-sys v0.4.1
  Downloaded dirs v5.0.1
  Downloaded futures-executor v0.3.30
  Downloaded futures-io v0.3.30
  Downloaded futures-macro v0.3.30
  Downloaded futures-sink v0.3.30
  Downloaded pin-utils v0.1.0
  Downloaded slab v0.4.9
  Downloaded percent-encoding v2.3.1
  Downloaded itoa v1.0.11
  Downloaded static_assertions v1.1.0
  Downloaded filedescriptor v0.8.2
  Downloaded signal-hook-mio v0.2.3
  Downloaded version_check v0.9.4
  Downloaded lru v0.12.3
  Downloaded paste v1.0.15
  Downloaded heck v0.5.0
  Downloaded rustversion v1.0.17
  Downloaded either v1.13.0
  Downloaded unicode-truncate v1.1.0
  Downloaded shell-words v1.1.0
  Downloaded lock_api v0.4.12
  Downloaded cfg-if v1.0.0
  Downloaded smallvec v1.13.2
  Downloaded parking_lot_core v0.9.10
  Downloaded signal-hook-registry v1.4.2
  Downloaded unicode-ident v1.0.12
  Downloaded autocfg v1.3.0
  Downloaded scopeguard v1.2.0
  Downloaded quote v1.0.36
  Downloaded serde_spanned v0.6.7
  Downloaded tinyvec_macros v0.1.1
  Downloaded strum v0.26.3
1 out of 1 hunk FAILED -- saving rejects to file src/lib.rs.rej
 Downloading crates ...
  Downloaded android-tzdata v0.1.1
  Downloaded urlencoding v2.1.3
  Downloaded android_system_properties v0.1.5
  Downloaded iana-time-zone v0.1.60
  Downloaded trash v5.0.0
  Downloaded chrono v0.4.38
   Compiling proc-macro2 v1.0.86
   Compiling unicode-ident v1.0.12
   Compiling libc v0.2.155
   Compiling autocfg v1.3.0
   Compiling version_check v0.9.4
   Compiling cfg-if v1.0.0
   Compiling rustversion v1.0.17
   Compiling lock_api v0.4.12
   Compiling memchr v2.7.4
   Compiling quote v1.0.36
   Compiling syn v2.0.71
   Compiling once_cell v1.19.0
   Compiling strsim v0.11.1
   Compiling ahash v0.8.11
   Compiling parking_lot_core v0.9.10
   Compiling anyhow v1.0.86
   Compiling heck v0.5.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling fnv v1.0.7
   Compiling syn v1.0.109
   Compiling ident_case v1.0.1
   Compiling proc-macro-error v1.0.4
   Compiling cc v1.1.6
   Compiling log v0.4.22
   Compiling signal-hook v0.3.17
   Compiling slab v0.4.9
   Compiling serde v1.0.204
   Compiling scopeguard v1.2.0
   Compiling smallvec v1.13.2
   Compiling num-traits v0.2.19
   Compiling paste v1.0.15
   Compiling pin-project-lite v0.2.14
   Compiling futures-core v0.3.30
   Compiling zerocopy v0.7.35
   Compiling either v1.13.0
   Compiling darling_core v0.20.9
   Compiling signal-hook-registry v1.4.2
   Compiling allocator-api2 v0.2.18
   Compiling crossbeam-utils v0.8.20
   Compiling itoa v1.0.11
   Compiling hashbrown v0.14.5
   Compiling bitflags v2.6.0
   Compiling parking_lot v0.12.3
   Compiling serde_derive v1.0.204
   Compiling futures-macro v0.3.30
   Compiling strum_macros v0.26.4
   Compiling tokio-macros v2.4.0
   Compiling darling_macro v0.20.9
   Compiling darling v0.20.9
   Compiling serde_json v1.0.121
   Compiling simd-adler32 v0.3.7
   Compiling derive_builder_core v0.20.0
   Compiling stability v0.2.1
   Compiling utf8parse v0.2.2
   Compiling ryu v1.0.18
   Compiling futures-sink v0.3.30
   Compiling anstyle-parse v0.2.4
   Compiling derive_builder_macro v0.20.0
   Compiling mio v0.8.11
   Compiling vergen-lib v0.1.3
   Compiling anstyle v1.0.7
   Compiling powerfmt v0.2.0
   Compiling adler v1.0.2
   Compiling thiserror v1.0.62
   Compiling regex-syntax v0.8.4
   Compiling colorchoice v1.0.1
   Compiling pkg-config v0.3.30
   Compiling is_terminal_polyfill v1.70.0
   Compiling anstyle-query v1.1.0
   Compiling anstream v0.6.14
   Compiling miniz_oxide v0.7.4
   Compiling deranged v0.3.11
   Compiling derive_builder v0.20.0
   Compiling futures-channel v0.3.30
   Compiling thiserror-impl v1.0.62
   Compiling clap_derive v4.5.11
   Compiling getset v0.1.2
   Compiling vergen v9.0.0
   Compiling aho-corasick v1.1.3
   Compiling socket2 v0.5.7
   Compiling mio v1.0.1
   Compiling equivalent v1.0.1
   Compiling pin-utils v0.1.0
   Compiling time-core v0.1.2
   Compiling tinyvec_macros v0.1.1
   Compiling bytes v1.6.1
   Compiling num-conv v0.1.0
   Compiling rustix v0.38.34
   Compiling futures-io v0.3.30
   Compiling num_threads v0.1.7
   Compiling clap_lex v0.7.1
   Compiling futures-task v0.3.30
   Compiling time v0.3.36
   Compiling clap_builder v4.5.11
   Compiling futures-util v0.3.30
   Compiling tokio v1.39.2
   Compiling regex-automata v0.4.7
   Compiling tinyvec v1.6.0
   Compiling indexmap v2.2.6
   Compiling signal-hook-mio v0.2.3
   Compiling itertools v0.13.0
   Compiling castaway v0.2.3
   Compiling vergen-gitcl v1.0.0
   Compiling unicode-segmentation v1.11.0
   Compiling percent-encoding v2.3.1
   Compiling static_assertions v1.1.0
   Compiling linux-raw-sys v0.4.14
   Compiling unicode-width v0.1.13
   Compiling unicode-truncate v1.1.0
   Compiling compact_str v0.7.1
   Compiling crossterm v0.27.0
   Compiling unicode-normalization v0.1.23
   Compiling clap v4.5.11
   Compiling strum v0.26.3
   Compiling lru v0.12.3
   Compiling num-integer v0.1.46
   Compiling cassowary v0.3.0
   Compiling option-ext v0.2.0
   Compiling bitflags v1.3.2
   Compiling home v0.5.9
   Compiling unicode-bidi v0.3.15
   Compiling minimal-lexical v0.2.1
   Compiling nom v7.1.3
   Compiling idna v0.5.0
   Compiling regex v1.10.5
   Compiling which v6.0.1
   Compiling dirs-sys v0.4.1
   Compiling ratatui v0.27.0
   Compiling num-bigint v0.4.6
   Compiling clap_complete v4.5.11
   Compiling form_urlencoded v1.2.1
   Compiling futures-executor v0.3.30
   Compiling toml_datetime v0.6.8
   Compiling serde_spanned v0.6.7
   Compiling crossbeam-epoch v0.9.18
   Compiling bstr v1.9.1
   Compiling crc32fast v1.4.2
   Compiling arrayvec v0.7.4
   Compiling aligned-vec v0.5.0
   Compiling rayon-core v1.12.1
   Compiling built v0.7.4
   Compiling winnow v0.6.16
   Compiling v_frame v0.3.8
   Compiling crossbeam-deque v0.8.5
   Compiling rav1e v0.7.1
   Compiling flate2 v1.0.30
   Compiling futures v0.3.30
   Compiling url v2.5.2
   Compiling num-rational v0.4.2
   Compiling dirs v5.0.1
   Compiling toml_edit v0.22.18
   Compiling luajit-src v210.5.8+5790d25
   Compiling validator_derive v0.18.1
   Compiling profiling-procmacros v1.0.15
   Compiling lua-src v546.0.2
   Compiling tracing-core v0.1.32
   Compiling typenum v1.17.0
   Compiling bytemuck v1.16.1
   Compiling shell-words v1.1.0
   Compiling yazi-shared v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-shared)
   Compiling mlua-sys v0.6.1
   Compiling validator v0.18.1
   Compiling profiling v1.0.15
   Compiling av1-grain v0.2.3
   Compiling toml v0.8.17
   Compiling globset v0.4.14
   Compiling clap_complete_nushell v4.5.3
   Compiling clap_complete_fig v4.5.2
   Compiling num-derive v0.4.2
   Compiling arg_enum_proc_macro v0.3.4
   Compiling itertools v0.12.1
   Compiling spin v0.9.8
   Compiling simd_helpers v0.1.0
   Compiling generic-array v0.14.7
   Compiling maybe-rayon v0.1.1
   Compiling quick-error v2.0.1
   Compiling arc-swap v1.7.1
   Compiling new_debug_unreachable v1.0.6
   Compiling bitstream-io v2.5.0
   Compiling noop_proc_macro v0.3.0
   Compiling weezl v0.1.8
   Compiling imgref v1.10.1
   Compiling loop9 v0.1.5
   Compiling yazi-config v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-config)
   Compiling flume v0.11.0
   Compiling rgb v0.8.45
   Compiling avif-serialize v0.8.1
   Compiling zune-inflate v0.2.54
   Compiling fdeflate v0.3.4
   Compiling ordered-float v2.10.1
   Compiling half v2.4.1
   Compiling bit_field v0.10.2
   Compiling zune-core v0.4.12
   Compiling color_quant v1.1.0
   Compiling byteorder-lite v0.1.0
   Compiling lebe v0.5.2
   Compiling typeid v1.0.0
   Compiling jpeg-decoder v0.3.1
   Compiling exr v1.72.0
   Compiling tiff v0.9.1
   Compiling erased-serde v0.4.5
   Compiling image-webp v0.1.3
   Compiling gif v0.13.1
   Compiling zune-jpeg v0.4.13
   Compiling ravif v0.11.9
   Compiling mlua_derive v0.9.3
   Compiling serde-value v0.7.0
   Compiling png v0.17.13
   Compiling rayon v1.10.0
   Compiling qoi v0.4.1
   Compiling tracing-attributes v0.1.27
   Compiling rustc-hash v2.0.0
   Compiling mutate_once v0.1.1
   Compiling tracing v0.1.40
   Compiling kamadak-exif v0.5.5
   Compiling image v0.25.2
   Compiling mlua v0.9.9
   Compiling yazi-boot v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-boot)
   Compiling ansi-to-tui v3.1.0
   Compiling onig_sys v69.8.1
   Compiling imagesize v0.13.0
   Compiling base64 v0.22.1
   Compiling yazi-adapter v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-adapter)
warning: yazi-boot@0.3.0: not within a suitable 'git' worktree!
warning: yazi-boot@0.3.0: VERGEN_GIT_COMMIT_DATE set to default
warning: yazi-boot@0.3.0: VERGEN_GIT_SHA set to default
   Compiling block-buffer v0.10.4
   Compiling crypto-common v0.1.6
   Compiling yazi-dds v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-dds)
   Compiling same-file v1.0.6
   Compiling walkdir v2.5.0
warning: yazi-dds@0.3.0: not within a suitable 'git' worktree!
warning: yazi-dds@0.3.0: VERGEN_GIT_COMMIT_DATE set to default
warning: yazi-dds@0.3.0: VERGEN_GIT_SHA set to default
   Compiling digest v0.10.7
   Compiling tokio-stream v0.1.15
   Compiling tokio-util v0.7.11
   Compiling uzers v0.12.0
   Compiling quick-xml v0.31.0
   Compiling lazy_static v1.5.0
   Compiling line-wrap v0.2.0
   Compiling base64 v0.21.7
   Compiling plist v1.6.1
   Compiling onig v6.4.0
   Compiling md-5 v0.10.6
   Compiling yazi-proxy v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-proxy)
   Compiling bincode v1.3.3
   Compiling concurrent-queue v2.5.0
   Compiling parking v2.2.0
   Compiling syntect v5.2.0
   Compiling event-listener v4.0.3
   Compiling tikv-jemalloc-sys v0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7
   Compiling backtrace v0.3.73
   Compiling inotify-sys v0.1.5
   Compiling instant v0.1.13
   Compiling overload v0.1.1
   Compiling gimli v0.29.0
   Compiling yazi-prebuild v0.1.2
   Compiling yazi-plugin v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-plugin)
   Compiling addr2line v0.22.0
   Compiling nu-ansi-term v0.46.0
   Compiling notify-types v1.0.0 (https://github.com/notify-rs/notify.git?rev=96dec74316a93bed6eec9db177b233e6e017275e#96dec743)
   Compiling inotify v0.10.2
   Compiling async-priority-channel v0.2.0
   Compiling sharded-slab v0.1.7
   Compiling tracing-log v0.2.0
   Compiling thread_local v1.1.8
   Compiling object v0.36.1
   Compiling filetime v0.2.23
   Compiling rustc-demangle v0.1.24
   Compiling notify v6.1.1 (https://github.com/notify-rs/notify.git?rev=96dec74316a93bed6eec9db177b233e6e017275e#96dec743)
   Compiling tracing-subscriber v0.3.18
   Compiling yazi-scheduler v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-scheduler)
error[E0433]: failed to resolve: use of undeclared crate or module `trash`
   --> yazi-scheduler/src/file/file.rs:143:14
    |
143 |                     Ok::<_, trash::Error>(())
    |                             ^^^^^ use of undeclared crate or module `trash`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `yazi-scheduler` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-08-01 12:21:23 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10197955758">10197955758</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/yazi/0001-fix-android-building-error-caused-by-trash-crate-139.patch
+++ b/packages/yazi/0001-fix-android-building-error-caused-by-trash-crate-139.patch
@@ -1,0 +1,25 @@
+From 9d1fab84efdd78dcf102262601d6965ec760ad95 Mon Sep 17 00:00:00 2001
+From: Xerxes-2 <dspxue@gmail.com>
+Date: Fri, 2 Aug 2024 14:46:43 +1000
+Subject: [PATCH] fix: android building error caused by `trash` crate (#1393)
+
+---
+ yazi-scheduler/src/file/file.rs | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/yazi-scheduler/src/file/file.rs b/yazi-scheduler/src/file/file.rs
+index 5b06385..24e8090 100644
+--- a/yazi-scheduler/src/file/file.rs
++++ b/yazi-scheduler/src/file/file.rs
+@@ -140,7 +140,7 @@ impl File {
+ 					{
+ 						trash::delete(&task.target)?;
+ 					}
+-					Ok::<_, trash::Error>(())
++					Ok::<_, anyhow::Error>(())
+ 				})
+ 				.await??;
+ 				self.prog.send(TaskProg::Adv(task.id, 1, task.length))?;
+-- 
+2.46.0
+

--- a/packages/yazi/build.sh
+++ b/packages/yazi/build.sh
@@ -25,7 +25,9 @@ termux_step_pre_configure() {
 }
 
 termux_step_make() {
-	YAZI_GEN_COMPLETIONS=true cargo build --jobs $TERMUX_PKG_MAKE_PROCESSES --target $CARGO_TARGET_NAME --release
+	VERGEN_GIT_SHA="termux" \
+	YAZI_GEN_COMPLETIONS=true \
+	cargo build --jobs $TERMUX_PKG_MAKE_PROCESSES --target $CARGO_TARGET_NAME --release
 }
 
 termux_step_make_install() {

--- a/packages/yazi/build.sh
+++ b/packages/yazi/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://yazi-rs.github.io/
 TERMUX_PKG_DESCRIPTION="Blazing fast terminal file manager written in Rust, based on async I/O"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.2.5"
+TERMUX_PKG_VERSION="0.3.0"
 TERMUX_PKG_SRCURL=https://github.com/sxyazi/yazi/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=aea4a6ebd7f56c5d5fe6afbea143fc98c336d4ccd7852f2e8983c23e205988c4
+TERMUX_PKG_SHA256=0a0c1583accca16759392f258367156a2c36fb0b1d37152b07e1aa5239c531ff
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 

